docs/api.md

```diff
@@ -3,4 +3,4 @@
 The most important parameter for all endpoints except the `Home#index` is the
```

Comment:
Any reason to allow passing token via params given the apparent security vulnerabilities?